release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1590,15 +1590,41 @@ Previously, the image pruner stopped when it failed to delete an image. As a res
 
 
 
-*OLM*
+*Operator Lifecycle Manager (OLM)*
 
 * Previously, `CustomResourceDefinition` (CRD) objects applied as part of an Operator installation could sometimes satisfy the installation requirements of a newer version of the same Operator. Consequently, during an Operator upgrade, the version being replaced could be prematurely removed. In some cases, the upgrade would stop. With this update, the CRDs that are created or updated as part of the Operator bundle installation are annotated to indicate their bundle of origin. These annotations are used by the `ClusterServiceVersion` (CSV) object to distinguish between pre-existing CRDs and same-bundle CRDs. As a result, upgrades will not complete until the current version's CRDs have been applied. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1947946[*BZ#1947946*])
 
-* Previously, pods that ran an index referenced by a `CatalogSource` object did not have `readOnlyRootFileSystem: false` explicitly set in their `securityContext` field. Consequently, if an SCC existed that enforced `readOnlyRootFileSystem: true` and matched the `securityContext` of the that pod, it would be assigned to that pod and cause it to fail repeatedly. This update explicitly sets `readOnlyRootFileSystem: false` in the `securityContext` field. As a result, pods that are referenced by a `CatalogSource` object are no longer matched to SCCs that enforce a read-only root filesystem and no longer fail. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1961472[*BZ#1961472*])
+* Previously, pods that ran an index referenced by a `CatalogSource` object did not have `readOnlyRootFileSystem: false` explicitly set in their `securityContext` field. Consequently, if a security context constraint (SCC) existed that enforced `readOnlyRootFileSystem: true` and matched the `securityContext` of the that pod, it would be assigned to that pod and cause it to fail repeatedly. This update explicitly sets `readOnlyRootFileSystem: false` in the `securityContext` field. As a result, pods that are referenced by a `CatalogSource` object are no longer matched to SCCs that enforce a read-only root file system and no longer fail. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1961472[*BZ#1961472*])
+
+* Because Operator Lifecycle Manager (OLM) does not allow skipped versions to be installed if specified in the `startingCSV` field during initial installations. This caused those skipped versions to be unable to be installed, even if users wanted to install them regardless of reasons why they are skipped. This fix updates OLM to allow users to install skipped versions only during initial installation by using the `startingCSV` specification in `Subscription` objects; users still cannot upgrade to skipped versions, as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1906056[*BZ#1906056*])
```

Comment:
s/Because/Because the ??
s/does not allow/did not allow @jeana-redhat is that right?
The first sentence seems like a fragment.

Reply:
Yea this was just a fragment goof. Changed to:

> Operator Lifecycle Manager (OLM) previously did not allow skipped versions to be installed if specified in the `startingCSV` field during initial installations.